.gitignore

```diff
@@ -4,3 +4,6 @@
 /dist-test
 /.idea
 /coverage
+/.vscode
+yarn-error.log
+yarn.lock
```

Comment:
Why ignore this? Don't we want the lockfile committed?

Reply:
Seeing as there was not mention of `yarn` and there was no existing lockfile, I didn't want to clutter my first PR with yarn specific files.